### DISCUSSION TITLE
Trying @auth/sveltekit, getting `Cannot read properties of undefined (reading 'AuthHandler')``

### DISCUSSION THREAD

**Discussion by janzheng** [nextauthjs/next-auth#6059] [Help] [answered]
Hi there!

I'm new to @auth and trying the sveltekit example app from the repo. Using a clean install from the repo, I'm getting `Cannot read properties of undefined (reading 'AuthHandler')`

I'm trying to understand if this is a config error on my part, or something wonky going on with the package?

**Comment by v-erse** [comment]
Same here! I even tried cloning the [example app](https://github.com/nextauthjs/sveltekit-auth-example), but I still get this error

Edit: Link to a repo where this error shows up https://github.com/v-erse/authjs-sveltekit

**Comment by balazsorban44** [comment, marked as answer]
Fixed here: https://github.com/nextauthjs/next-auth/releases/tag/%40auth%2Fsveltekit%40v0.1.6


The underlying issue is tracked here: https://github.com/nextauthjs/next-auth/issues/6068
